src/webserver/http.rs

```diff
@@ -343,8 +343,12 @@ impl SingleOrVec {
 }
 
 /// Resolves the path in a query to the path to a local SQL file if there is one that matches
-fn path_to_sql_file(path: &str) -> Option<PathBuf> {
-    let mut path = PathBuf::from(path.strip_prefix('/').unwrap_or(path));
+async fn path_to_sql_file(req: &ServiceRequest, path: &str) -> Option<PathBuf> {
+    let app_state: &web::Data<AppState> = req.app_data().expect("app_state");
+    let strip_path = path
+        .strip_prefix(&app_state.config.site_prefix)
```

Comment:
If we ensured the site prefix is '/' by default and always ends with '/' we could avoid the repeated strip_prefix calls here.

Reply:
I tried that initially and had some trouble... but then again even what I had now wasn't working at some point. So I will try again because I agree it makes more sense that way.

Reply:
Fixed in 013eb8f